tests/output-normalization.sh

```diff
@@ -0,0 +1,9 @@
+source common.sh
+
+testNormalization () {
```

Comment:
This test also do not yet work with CA derivations, so I pulled it out into its own file.